mappings/net/minecraft/util/Int2ObjectLinkedHashMap.mapping

```diff
@@ -0,0 +1,30 @@
+CLASS net/minecraft/class_80 net/minecraft/util/Int2ObjectLinkedHashMap
```

Comment:
I think you mean CrudeIncrementalIntIdentityHashBiMap

Reply:
Does it even behave like a linked hash bi map though

Reply:
I haven't checked the code

Reply:
It functions as a hash linked hash map